### PR TITLE
Feature/34 program detail UI component

### PR DESCRIPTION
**Pull request by KoglerT**
# 💻 What
See if page looks like UX wanted
#34 

# ❓ Why
TODO

# 📘 How
TODO

# 👀 See
| Light Mode | Dark Mode |
| --- | --- |
|  |  |

### CODE REVIEW

**Review by dev-Fabi — APPROVED** [review]
Nice, I just extracted the component into its own composable and added some fixes for content overflow / image scaling.

File: app/src/main/java/at/irfc/app/ui/program/details/EventDetailCard.kt [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            if (event.additionalImages.isNotEmpty()) {
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(8.dp)
+                        .requiredHeight(100.dp),
+                    horizontalArrangement = Arrangement.SpaceEvenly
+                ) {
+                    event.additionalImages.forEach { image ->
+                        // TODO define a image ration
```
Comment (kaulex99) [inline app/src/main/java/at/irfc/app/ui/program/details/EventDetailCard.kt:81]: Want to fix todo?

**Review by kaulex99 — APPROVED** [review]
LGTM 🚀

File: app/src/main/java/at/irfc/app/ui/program/details/EventDetailCard.kt (hunk 1 above)
Comment (KoglerT) [reply app/src/main/java/at/irfc/app/ui/program/details/EventDetailCard.kt:81]: > Want to fix todo?

Der Fabi wollte das mit dir Besprechen

**Pull request merged**